scripts/train/train.py

```diff
@@ -387,6 +387,10 @@ def main(cfg: DictConfig) -> Trainer:
                                                        'load_ignore_keys',
                                                        must_exist=False,
                                                        default_value=None)
+    compile_config: Optional[Dict[str, Any]] = pop_config(cfg,
+                                                          'compile_config',
+                                                          must_exist=False,
+                                                          default_value=None)
```

Comment:
@dakinggg do we want to add compile configs yet?

@crinard did compile help in any settings?

Reply:
if it helped at all, sure, why not?

Reply:
iirc there were maybe one/two places where it helped slightly, although the more common result of a non-empty compile config was errors.

Reply:
ok lets keep it in case people want to use it in the future.